core/src/main/scala/org/apache/spark/eventhubs/EventHubsConf.scala

```diff
@@ -355,6 +355,10 @@ final class EventHubsConf private (private val connectionStr: String)
    * @return the updated [[EventHubsConf]] instance
    */
   def setReceiverTimeout(d: Duration): EventHubsConf = {
+    if (d.toMillis > operationTimeout.getOrElse(DefaultOperationTimeout).toMillis) {
```

Comment:
We should add a similar check in the setOperationTimeout as well to cover the case when the user just updates the OperationTimeout and sets it to a value less than the receiver timeout.